core/datastore/src/test/kotlin/com/google/samples/apps/nowinandroid/core/datastore/NiaPreferencesDataSourceTest.kt

```diff
@@ -16,7 +16,7 @@
 
 package com.google.samples.apps.nowinandroid.core.datastore
 
-import com.google.samples.apps.nowinandroid.core.datastore.test.testUserPreferencesDataStore
+import com.google.samples.apps.nowinandroid.core.datastore.di.testUserPreferencesDataStore
```

Comment:
Is this package of code for the special workaround for KSP and test fixtures?

Reply:
> the special workaround for KSP

not sure what you are referring to.

I did this renaming simply because `TestDataModule` is a "fixture" of `DataModule`, which is defined in `com.google.samples.apps.nowinandroid.core.datastore.di`.

I re-use the same package for 2 main reasons:
- there is no added value of creating new packages (to me, adding "test" is not helping in any way, but if we were to always add "test", then it should be `[...].core.datastore.di.test`)
- keeping the same package makes it use less import statements

Reply:
It was not about removing `test` but adding `di`. From the whole PR, I saw the `di` package responsible for Hilt test modules. Here is the function import, and it looks like it is out of pattern.

Reply:
`testUserPreferencesDataStore` was declared in the same file as `TestDatastoreModule` so I kept it in the same file. But you are right, it has technically nothing to do in the"di" package, as it is also used in other places.

Reply:
Clear!